packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.test.js

```diff
@@ -49,130 +61,127 @@ describe('<OperationTableDetails>', () => {
-    expect(wrapper).toMatchSnapshot();
+    const { container } = render(
+      <OperationTableDetails {...props} data={[cloneServiceOpsMetrics]} loading={false} />
+    );
+    expect(container.querySelector('table')).toBeInTheDocument();
   });
 
   it('render P95 latency with more than 2 decimal places value', () => {
-    const cloneServiceOpsMetrics = {};
-    Object.assign(cloneServiceOpsMetrics, serviceOpsMetrics[0]);
+    const cloneServiceOpsMetrics = { ...serviceOpsMetrics[0] };
     cloneServiceOpsMetrics.latency = 0.2988;
-    wrapper.setProps({ ...props, data: [cloneServiceOpsMetrics], loading: false });
-    expect(wrapper).toMatchSnapshot();
+    const { container } = render(
+      <OperationTableDetails {...props} data={[cloneServiceOpsMetrics]} loading={false} />
+    );
+    expect(container.querySelector('table')).toBeInTheDocument();
   });
 
   it('test column render function', () => {
-    wrapper.setProps({
-      ...props,
-      data: [
-        {
-          ...serviceOpsMetrics,
-          dataPoints: {
-            ...serviceOpsMetrics.dataPoints,
-            service_operation_call_rate: [],
-            service_operation_error_rate: [],
-            service_operation_latencies: [],
-          },
+    const data = [
+      {
+        ...serviceOpsMetrics[0],
+        dataPoints: {
+          ...serviceOpsMetrics[0].dataPoints,
+          service_operation_call_rate: [],
+          service_operation_error_rate: [],
+          service_operation_latencies: [],
         },
-      ],
-      loading: false,
-    });
-    expect(wrapper).toMatchSnapshot();
+      },
+    ];
+    const { container } = render(<OperationTableDetails {...props} data={data} loading={false} />);
+    expect(container.querySelector('table')).toBeInTheDocument();
   });
 
   it('highlight the row', () => {
-    wrapper.setProps({ ...props, data: serviceOpsMetrics, loading: false });
-    expect(wrapper.state('hoveredRowKey')).toBe(-1);
+    render(<OperationTableDetails {...props} data={serviceOpsMetrics} loading={false} />);
 
-    wrapper.find('.table-row').at(0).simulate('mouseenter');
-    expect(wrapper.state('hoveredRowKey')).toBe(0);
+    const tableRow = screen.getAllByRole('row')[1];
 
-    wrapper.find('.table-row').at(0).simulate('mouseleave');
-    expect(wrapper.state('hoveredRowKey')).toBe(-1);
-  });
+    fireEvent.mouseEnter(tableRow);
 
-  it('highlight the row', () => {
-    wrapper.setProps({ ...props, data: serviceOpsMetrics, loading: false });
-    expect(wrapper.state('hoveredRowKey')).toBe(-1);
+    expect(screen.getByText('View traces')).toBeInTheDocument();
 
-    wrapper.find('.table-row').at(0).simulate('mouseenter');
-    expect(wrapper.state('hoveredRowKey')).toBe(0);
+    fireEvent.mouseLeave(tableRow);
 
-    wrapper.find('.table-row').at(0).simulate('mouseleave');
-    expect(wrapper.state('hoveredRowKey')).toBe(-1);
+    expect(screen.queryByText('View traces')).not.toBeInTheDocument();
```

Comment:
how does this test "highlights the row"? Should we be detecting some style change?

What's the equivalent of previous
```
expect(wrapper.state('hoveredRowKey')).toBe(0);
```